Assets/Scripts/Model/Edge.cs

```diff
@@ -8,15 +8,15 @@ namespace Assets.Scripts.Model
   [Serializable]
   public class Edge : IEquatable<Edge>
   {
+    public int id;
+    public string label;
     public int from;
+    public int to;
+    public int weight;
 
     [NonSerialized] public GameObject gameObject;
-    public int id;
-    public string label;
     [NonSerialized] public Node nodeFrom;
     [NonSerialized] public Node nodeTo;
-    public int to;
-    public int weight;
 
```

Comment:
I feel like the ordering of attributes was fixed like 10 times now. I think smth got messed up in merges xd

Reply:
maybe, this one will fix it for good